jfr-streaming/src/main/java/io/opentelemetry/contrib/jfr/metrics/internal/network/NetworkWriteHandler.java

```diff
@@ -23,18 +38,30 @@
 //        ]
 // }
 
-import io.opentelemetry.api.metrics.Meter;
-import io.opentelemetry.contrib.jfr.metrics.internal.AbstractThreadDispatchingHandler;
-import io.opentelemetry.contrib.jfr.metrics.internal.RecordedEventHandler;
-import io.opentelemetry.contrib.jfr.metrics.internal.ThreadGrouper;
-
 public final class NetworkWriteHandler extends AbstractThreadDispatchingHandler {
-  static final String EVENT_NAME = "jdk.SocketWrite";
-  private final Meter otelMeter;
+  private static final String EVENT_NAME = "jdk.SocketWrite";
+  private static final String METRIC_NAME_DURATION = "runtime.jvm.network.write.duration";
+  private static final String METRIC_NAME_BYTES = "runtime.jvm.network.write.io";
+  private static final String DESCRIPTION_BYTES = "Bytes Written";
+  private static final String DESCRIPTION_DURATION = "Write Duration";
```

Comment:
This is likely the issue between duration read.  This needs to be a more generic description about duration of a network operation.

Reply:
Cool yeah that did the trick 👍